gnocchi/chef.py

```diff
@@ -92,3 +97,41 @@ def expunge_metrics(self, sync=False):
                             raise
                         LOG.error("Unable to expunge metric %s from storage",
                                   metric, exc_info=True)
+
+    def refresh_metric(self, metric, timeout):
```

Comment:
for discussion, any reason why you didn't put refresh_metric in api as it's the only place it's used?

Reply:
Hm that requires knowledge about locking, sacks, etc, so it seems good to be there.